plugin/trino-hive/src/test/java/io/trino/plugin/hive/TestHivePageSink.java

```diff
@@ -132,6 +141,14 @@ public void testAllFormats()
         }
     }
 
+    private boolean isSupportedCodec(HiveStorageFormat storageFormat, HiveCompressionOption compressionOption)
```

Comment:
nit: in `BaseHiveConnectorTest` you have this inline

Reply:
Yeah - in a similar one. Will structure same way.